examples/react-todos/.meteor/packages

```diff
@@ -13,5 +14,6 @@ reactive-var
 meteorhacks:npm
 cosmos:browserify
 
+static-html
```

Comment:
Yes I think it should be inside.
Right now we use `ng.html` to minify and also add the template to `template-cache` but it will be better that it will be optional.
Developers could use regular HTML files and if they really want they would use `ng.html` for template cache. Right now everything is in template cache.

By the way, what about minification and other plugins?

https://github.com/Urigo/angular-meteor/blob/master/plugin/handler.js